package.json

```diff
@@ -180,6 +187,16 @@
           "type": "boolean",
           "default": false,
           "description": "Only update compiler diagnostics on save, not on document change."
+        },
+        "elmTestRunner.logpanel": {
+          "description": "write diagnotic logs to an output panel",
```

Comment:
typo

Reply:
Actually, I' going to remove this setting, as I think the logpanel should always be shown